src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -231,6 +231,7 @@ const ReleaseDataUploadsSection = ({
       <LoadingSpinner loading={isLoading}>
         {dataFiles.length > 0 ? (
           <ReorderableAccordion
+            canReorder={canUpdateRelease}
```

Comment:
Just noticed this, but given that we're reordering files, shouldn't this have `reorderHiddenText="files"`?